app/models/katello/product_content.rb

```diff
@@ -3,6 +3,8 @@ class ProductContent < Katello::Model
     belongs_to :product, :class_name => 'Katello::Product', :inverse_of => :product_contents
     belongs_to :content, :class_name => 'Katello::Content', :inverse_of => :product_contents
 
+    validates :enabled, inclusion: { in: [true, false], message: N_("must be true or false") }
```

Comment:
Added this because what if `set_enabled_from_candlepin!` comes back with a `nil`..